exonum-java-binding-fakes/src/main/java/com/exonum/binding/fakes/services/service/TestService.java

```diff
@@ -42,12 +43,24 @@
 
   private final SchemaFactory<TestSchema> schemaFactory;
 
+  private long blockchainHeight;
+
+  private int validatorId;
+
   @Inject
   public TestService(SchemaFactory<TestSchema> schemaFactory) {
     super(ID, NAME, (rawTx) -> PutValueTransaction.from(rawTx, schemaFactory));
     this.schemaFactory = schemaFactory;
   }
 
+  public long getBlockchainHeight() {
```

Comment:
@vitvakatu, @skletsun Are such methods accessible in native (can you implement tests for afterCommit with these)?

If not, please suggest what will help.

Reply:
As for me - looks like it's enough for testing from the native side.